### DISCUSSION TITLE
Dashboard is empty

### DISCUSSION THREAD

**Discussion by converter-user-old** [authorizerdev/authorizer#226] [General]
Hi,

what did i do wrong when the Dashboard ist empty but the Docker Container runs without problems.
In the devtools i can see that the request are unauthorized but i logged in with the right admin password ( [{message: "unauthorized", path: ["_add_email_template"]}] )

![image](https://user-images.githubusercontent.com/85690286/188611763-610aacaa-0732-4203-9251-a494455a9fa9.png)

**Comment by lakhansamani** [comment]
@ruessej can you share some system logs?
That could help me debug further 

thanks

**Reply by converter-user-old** [reply]
Here are my logs from docker: https://gist.github.com/ruessej/f606453f018a098fd615349408f3b1a9

**Reply by lakhansamani** [reply]
Thank you for sharing this,
Logs seem okay to me,

It seems there is some failure point that is not logged.
May be we can get on call and resolve this :-)

You can send me an invite on lakhan.m.samani@gmail.com for today
